modules/minimum-required-permissions-ipi-gcp-provided-sas.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="minimum-required-permissions-ipi-gcp-provided-sas_{context}"]
+= Required GCP permissions for user-provided service accounts
+
+When you are installing	a cluster, the compute and control plane nodes require their own service accounts.
+The default scenario occurs when the user does not provide a service account email address for the control plane or compute nodes in the `install-config.yaml` file.
+In this case the installation program will create a service account for the control plane and compute nodes, requiring additional permissions.
+For more information, see the _Installation configuration parameters for GCP_ page.
```

Comment:
Trying to orient this more around the use case, maybe something like this?

```suggestion
When you are installing a cluster, the compute and control plane nodes require their own service accounts.
By default, the installation program creates a service account for the control plane and compute nodes.
This service account requires the roles and permissions that are listed in <whatever module has these>, as well as the `resourcemanager.projects.getIamPolicy` and `resourcemanager.projects.setIamPolicy` permissions.

If this approach does not meet the security requirements of your organization, you can provide a service account email address for the control plane or compute nodes in the `install-config.yaml` file.
For more information, see the _Installation configuration parameters for GCP_ page.
```

Reply:
I'm incorporating this suggestion, except line 11 will reference the installer service account, rather than the created service accounts. Thanks

Reply:
Update LGTM :rocket: